tracing-subscriber/src/registry/mod.rs

```diff
@@ -250,18 +264,42 @@ where
         })
     }
 
-    /// Returns an iterator over all parents of this span.
+    /// Returns an iterator over all parents of this span, starting with the
+    /// immediate parent.
     ///
     /// The iterator will first return the span's immediate parent, followed by
     /// that span's parent, followed by _that_ span's parent, and so on, until a
     /// it reaches a root span.
-    pub fn parents(&self) -> Parents<'_, R> {
+    pub fn parents(&self) -> Parents<'a, R> {
```

Comment:
For my education: why the switch from `'_` to `'a`?

Reply:
IIRC the other lifetime changes made this necessary, to prove that the iterator would live as long as the reference to the registry that produced it. Otherwise, the inferred lifetime is that of the borrow of `&self`, which is wrong.